auction-server/src/state.rs

```diff
@@ -1092,6 +1092,13 @@ impl Store {
         };
     }
 
+    pub fn broadcast_svm_chain_update(&self, update: SvmChainUpdate) {
+        match self.event_sender.send(UpdateEvent::SvmChainUpdate(update)) {
```

Comment:
While we are not doing anything in the `Ok` arm, we can use `if let Err(error) = ...`